ethereum/stratum/src/test/java/org/hyperledger/besu/ethereum/stratum/StratumConnectionTest.java

```diff
@@ -168,46 +174,9 @@ public void testStratum1SubmitHashrate() {
                 + "  \"params\": [ "
                 + "    \"0x03\",\"0x02\" "
                 + "  ]"
-                + "}\n"));
-    assertThat(called.get()).isFalse();
-
-    assertThat(message.get()).isEqualTo("{\"jsonrpc\":\"2.0\",\"id\":23,\"result\":true}\n");
-  }
-
-  @Test
-  public void testHttpMessage() {
```

Comment:
I'd like to see the exact same test work before and after to make sure there's no regression. Is that possible?

Reply:
This should be covered by `StratumServerTest#handleHttpPost`, I can add the `Connection` and `Accept-Encoding` headers though.

Reply:
I added both headers to the mentioned test